README.md

```diff
@@ -1,13 +1,90 @@
+- Get more reps and add to toolbox and tying concepts together
+- Work on teamwork and remote collaboration skills
+
+#### Why was this project created?
+- This project was created to teach the fundamentals of working on a team and implementing vanilla JavaScript to create a basic web application
+
+#### How does this work?
+![homepage screenshot](link in the parens)
+- Each page load will automatically generate a random motivational poster. From here, the user has options!
+- The user may continue to generate random posters by clicking the `Show Another Random Poster Button`.
+- To create their own poster, a user will click on `Make Your Own Poster`. This will pull up a form that accepts inputs for an image URL, a title, and a quote. Clicking `Show my poster` will return the user to the home page and display their new poster! There is also an option to return to the home page without creating a poster, by clicking `Nevermind, take me back!`
+- Posters can also be stored for later viewing by clicking `Save This Poster`. This switches to a grid view displaying mini versions of all saved posters.
+![saved posters view + delete gif](link in the parens)
+- To delete a poster, the user can double click the image.
+- This applications runs directly in the web browser!
+
+#### Want to contribute?
+- Interested in contributing to Hang in there?!
+- Fork this project and clone to your machine.
+- Create a pull request, and one of our project managers will get back to you with a code review!
+
+#### Technologies Used
+- JavaScript
+- HTML
+- CSS
+
+#### Project Reflection
+##### Nichele
+- I had a great experience during this project. Peter and I spent a good deal of time planning and discussing the problems, challenges we might face, and how to address problems when encountered.
+##### Peter
+
+##### Wins
+##### Nichele
+- I felt like our communication styles worked well together, and we were able to address coding challenges through discussion and solo pseudocoding.
+- Planning! We spent a lot of time in discussion deciding how to address a problem before writing any code. This seemed to give us a clear direction without leaving us muddled in issues we hadn't accounted for.
+- Our git workflow seemed to come together pretty quickly and I didn't have any issues with merges or pulls.
+- We were able to reach out to mentors when we needed assistance, and were able to come up with a solution to our problem with some gentle nudging.
+##### Peter
+
+##### Challenges
+##### Nichele
+- I wish I had taken more notes! On the next project, I think I'll create a separate file within the project to store reflections and questions that come up during the project.
+- We didn't really utilize our mentors for code reviews. Though we did discuss issues we were having, we didn't actually add them to any of the PR code reviews. Next time I would like to discuss a code review schedule with my mentors prior to beginning the project.
+##### Peter
+
```

Comment:
- I too wish I'd taken more notes, or even some sort "progress document" to track the exact points of frustration and how exactly we were able to overcome these specific difficulties.  This could be useful in future projects, to be able to recall the methodologies that worked and use them again when appropriate.
- While I did reach out to my rock and mentor for reviews, I think it was a bit too close to the deadline to reasonably expect that.  In the future I agree that it would be wise to communicate with mentors ahead of time, establish a schedule, and allow them plenty of time for them to compete reviews and for the team to use that feedback to refactor accordingly.

Reply:
√